examples/policy-binding-sample-2.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: monokle.com/v1alpha1
+kind: MonoklePolicyBinding
+metadata:
+  name: policy-binding-sample-2
+spec:
+  policyName: "policy-sample-2"
+  validationActions: [Warn]
+  matchResources:
+    namespaceSelector:
+      matchLabels:
+        namespace: default
```

Comment:
I'm not entirely sure about this part. It follow what VAP has but it's labels matcher. Initially I thought we would have a property to define namespace name for which policy is applied (through this binding). This one matches `namespace` label in resource metadata which is not required and can have any value AFAIU.

I went through [the specs](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.28/#matchresources-v1beta1-admissionregistration-k8s-io) but haven't found something which would just match namespace for the resource 🤔

Reply:
also need to be mindful that no all resources are namespaced - is it possible to match on resource kind/version?

Reply:
This is optional part of MonoklePolicyBinding so skipping it will not restrict policy to be applied to single namespace (so will be applied cluster-wide).

As for more precise matching, we can follow [`matchResources` spec](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.28/#matchresources-v1beta1-admissionregistration-k8s-io) which allows filtering by resource, apiVersion, apiGroups, etc, with [`resourceRules`](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.28/#namedrulewithoperations-v1beta1-admissionregistration-k8s-io). By "following" I mean we need to implement matching mechanism according to specs (or part of it) in admission controller.